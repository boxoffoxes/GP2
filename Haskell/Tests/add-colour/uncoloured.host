[
	(n1, empty)
|
	(e1, n1, n1, empty)
]
