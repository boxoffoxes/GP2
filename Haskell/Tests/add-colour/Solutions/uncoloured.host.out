[
 (n1, empty # red)
|
 (e2, n1, n1, empty # dashed)
]
